site2/docs/io-cdc-debezium.md

```diff
@@ -92,7 +92,7 @@ You can use one of the following methods to create a configuration file.
         database.history: "org.apache.pulsar.io.debezium.PulsarDatabaseHistory"
         database.history.pulsar.topic: "history-topic"
         database.history.pulsar.service.url: "pulsar://127.0.0.1:6650"
-
+        pulsar.service.url: "pulsar://127.0.0.1:6650"
```

Comment:
I don't find any code to read `pulsar.service.url`, could you provide this code? The `database.history.pulsar.service.url` is correct.

Reply:
Yes, i just checked the code and also not found the `pulsar.service.url`. In my test i used the sample YAML on https://pulsar.apache.org/docs/en/io-debezium-source/#configuration-1 but functions gave me an ERROR about the missing config of `pulsar.service.url` then i noticed the following sample 
```
bin/pulsar-admin source localrun \
--archive connectors/pulsar-io-debezium-mysql-2.9.1.nar \
--name debezium-mysql-source --destination-topic-name debezium-mysql-topic \
--tenant public \
--namespace default \
--source-config '{"database.hostname": "localhost","database.port": "3306","database.user": "debezium","database.password": "dbz","database.server.id": "184054","database.server.name": "dbserver1","database.whitelist": "inventory","database.history": "org.apache.pulsar.io.debezium.PulsarDatabaseHistory","database.history.pulsar.topic": "history-topic","database.history.pulsar.service.url": "pulsar://127.0.0.1:6650","key.converter": "org.apache.kafka.connect.json.JsonConverter","value.converter": "org.apache.kafka.connect.json.JsonConverter","pulsar.service.url": "pulsar://127.0.0.1:6650","offset.storage.topic": "offset-topic"}'
```
Used the config of `"pulsar.service.url": "pulsar://127.0.0.1:6650"` and i also added it then passed the test.

I will do the test again to verify it.